swabbie-aws/src/main/kotlin/com/netflix/spinnaker/swabbie/aws/autoscalinggroups/Rules.kt

```diff
@@ -66,13 +68,14 @@ class ZeroInstanceDisabledServerGroupRule(
 class ZeroInstanceInDiscoveryDisabledServerGroupRule(
   private val discoveryClient: Optional<DiscoveryClient>,
   private val clock: Clock
-) : Rule<AmazonAutoScalingGroup> {
+) : Rule {
 
   @Value("\${swabbie.resource.disabled.days.count:30}")
   private val disabledDurationInDays: Long = 30
```

Comment:
will this ever change between the rules ? can we have a global value instead ?

Reply:
Yes, so this feature would allow to specify params. This rule will likely be replaced with smaller rules such as: `DiscoveryDisabledServerGroupRule`, `ZeroInstanceRule`, `DisabledLoadBalancerRule`:

```
enabledRules:
- operator: OR
    description: Empty Server Groups that have been disabled for 30 days
    rules:
    - name: ZeroInstanceRule
    - name: DisabledLoadBalancerRule
      parameters:
        longThanDays: 30
    - name: DiscoveryDisabledServerGroupRule
      parameters:
        longThanDays: 30
```